ou_dedetai/utils.py

```diff
@@ -85,9 +85,10 @@ def clean_all():
 
 def get_user_downloads_dir() -> str:
     home = Path.home()
+    downloads_path = str(home) # no guarantee that ~/Downloads exists (e.g. arch)
```

Comment:
Let's revert this change, and handle this in another PR.

Reply:
I don't mind doing that, but is it really necessary to split it out? The function still returns the user's `~/Downloads` folder if it's listed in `$XDG_CONFIG_HOME/user-dirs.dirs` or `$HOME/.config/user-dirs.dirs`. It's just the last-resort fallback that has changed.

Reply:
Idk about all users, but I take great care in organizing my top level home directory. I'd much rather even loose these caches files on reboot than have them polite my home-directory

Reply:
We could fallback to `/tmp`, I suppose. But this function is now only defining the user's folder to *search* before initiating a new download. If we decide to initiate the download, then it will go to the new cache location instead of `~/Downloads` or `~`; i.e. (`config.py` L999+):
```python
    @property
    def download_dir(self) -> str:
        if self._download_dir is None:
            self._download_dir = constants.DEFAULT_CACHE_DIR
        return self._download_dir
```

Reply:
Would rather revert then deal with this later